src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -544,4 +547,17 @@ public sealed class ProxyConfiguration
         public string BypassList { get; set; }
         public bool BypassOnLocal { get; set; }
     }
+
+    [Serializable]
+    public sealed class ExportCommandConfiguration
+    {
+        public ExportCommandConfiguration()
+        {
+            OutputFilePath = "packages.config";

```

Comment:
Do we do this other places?

Reply:
Yip, in at least one other place: https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs#L410

Reply:
Since this is done elsewhere, I think we are good to do this here as well.